db/migrate/20180312205753_create_wheres.rb

```diff
@@ -0,0 +1,12 @@
+class CreateWheres < ActiveRecord::Migration
+  def change
+    create_table :wheres do |t|
+      t.integer :user_id
+      t.datetime :sent_at
+      t.text :body
+      t.string :provenance
```

Comment:
What's this one for?

Reply:
For recording whether the status came from email or Mattermost